azurerm_iothub - support for the `endpoint.resource_group` property (#8032)
terraform-providers/terraform-provider-azurerm: website/docs/r/iothub.html.markdown

* `connection_string` - (Required) The connection string for the endpoint.

> * `endpoint_resource_group_name` - (Required) The name of the resource group of the endpoint.

Reviewer: ```suggestion
* `endpoint_resource_group_name` - (Required) The resource group in which the endpoint will be created.
```

Author: updated
